src/ripple/nodestore/impl/codec.h

```diff
@@ -494,16 +494,6 @@ nodeobject_compress (void const* in,
     std::pair<void const*, std::size_t> result;
     switch(type)
     {
-    case 0: // uncompressed
```

Comment:
It might be nice to leave a comment for why `case 0:` doesn't need to be covered.  `nodeobject_decompress` currently handles `case 0` as uncompressed.  Your change leaves an asymmetry between `nodeobject_compress` and `nodeobject_decompress`.